### DISCUSSION TITLE
init ratelimiter backend from config, not code

### DISCUSSION THREAD

**Discussion by ponytailer** [abersheeran/asgi-ratelimit#53] [Ideas]
For example:
```
RateLimiter(
    backend="simple/ratelimiter.backend.simple",
    ...
)
```

Because, different env config has the  same code.

In CI/CD,  use the simple, but  use the redis in production.

@abersheeran

**Comment by abersheeran** [comment]
I don't think it's too difficult for users to write their own code to read the configuration.🤔

**Reply by ponytailer** [reply]
so, one code supports two env and config. How?

**Reply by abersheeran** [reply]
"less is more"
